test/binCases/watch/info-verbosity-off/info-verbosity-off.test.js

```diff
@@ -1,13 +1,41 @@
 "use strict";
 
-jest.setTimeout(10E6);
+jest.setTimeout(10e6);
 /* eslint-disable node/no-unsupported-features  */
 /* eslint-disable node/no-unsupported-features/es-syntax  */
 
-const { runWatch } = require("../../../testUtils");
+const { extractSummary, extractHash, appendDataIfFileExists, runAndGetWatchProc } = require("../../../testUtils");
+const fs = require("fs");
+const path = require("path");
 
-test("info-verbosity-off", async(done) => {
-	const result = await runWatch(__dirname, [
+const fileToChange = "index.js";
+const copyFile = "index_copy.js";
+const fileToChangePath = path.resolve(__dirname, fileToChange);
+const copyFilePath = path.resolve(__dirname, copyFile);
+
+// create copy of "index.js" => "index_copy.js"
+beforeEach(() => {
+	// fs.copyFileSync was added in Added in: v8.5.0
```

Comment:
we still support node v.6 so you will need to make a workaround for this

Reply:
This is already a workaround. I did not use >=v8.5.0 feature `fs.copyFileSync` as we still support v6.

Reply:
ok nice